src/main/java/com/springBoot/project/mysqlConnect/ConnectionManager.java

```diff
@@ -0,0 +1,39 @@
+package com.springBoot.project.mysqlConnect;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+
+public class ConnectionManager {
+	private static final Logger logger = Logger.getLogger(ConnectionManager.class);
+	@Value("${jdbc.url}") private static String URL;
```

Comment:
Are you sure that it works `@Value` into `static` field? I suppose - no, because you can't read data from property file into `static` variable directly, the only way is a creating setter.
Additional info: https://stackoverflow.com/questions/30435672/how-to-inject-property-value-using-value-into-static-fields